packages/ui/src/components/Button/Button.tsx

```diff
@@ -83,7 +83,7 @@ export const Button: PolymorphicComponent = forwardRef(function Button<
     >
       <span className={childrenWrapper} style={{ opacity: loading ? 0 : 1 }}>
         {iconPosition === 'left' && Icon}
-        {children && <span className={textWrapper}>{children}</span>}
+        {children && <span className={clsx(textWrapper)}>{children}</span>}
```

Comment:
nit: do we need `clsx` here? 
```suggestion
        {children && <span className={textWrapper}>{children}</span>}
```

Reply:
Nope, probably a left over, will remove!